### DISCUSSION TITLE
How does foreign key relationships work?

### DISCUSSION THREAD

**Discussion by Dorfeuheinz** [vinniefranco/exandra#6] [Q&A]
I wanted to know how to model a foreign key relationship between two tables. I do know that denormalisation etc. is an option, but how do I map the relationship to a schema.

For example- I have a schema that looks like this-
```
schema "users" do
   field :name, :string
   field :age, :int 

   has_one :settings, Myapp.Accounts.Settings
end
```
& i create a table- 
`create table users(user_id uuid, name string, age integer, PRIMARY KEY(user_id)) WITH CLUSTERING;`

now if I have a settings table that corresponds to an indivisuals user account, how do I connect the 2 & what would the schema look like? Will the above schema even work?

**Comment by Dorfeuheinz** [comment]
found it sorry for the trouble

**Reply by Himanshu21git** [reply]
@Dorfeuheinz Could you please post the fix that you found as well.
